### DISCUSSION TITLE
Add shortcode as a dynamic tag in display conditions experiment

### DISCUSSION THREAD

**Discussion by rhmkt** [elementor/elementor#27298] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

The new Display Conditions experiment does not allow you to choose or enter a shortcode as dynamic data to compare against. In my case, I have a custom post type (Projects) with multiple custom fields. Each user then also has a custom field (Project ID). The shortcode then uses the Project ID to query which custom post (project) to grab data from and assign to various locations on the home page when logged in as a user. If the data is empty (or not complete, etc) the shortcode returns 0. So the display conditions would only show the field when the value is not 0, etc, etc.

### Describe the solution you'd like.

Add the ability to query a shortcode as a display condition.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by vgstef** [comment]
Great idea! And shortcodes could be used to test with Pods fields. Otherwise there is no way to use Pods in display conditions.
